aspnetcore/security/enforcing-ssl.md

```diff
@@ -39,3 +59,34 @@ For more information, see [URL Rewriting Middleware](xref:fundamentals/url-rewri
 
 Requiring HTTPS globally (`options.Filters.Add(new RequireHttpsAttribute());`) is a security best practice. Applying the
 `[RequireHttps]` attribute to all controllers/Razor Pages isn't considered as secure as requiring HTTPS globally. You can't guarantee the `[RequireHttps]` attribute is applied when new controllers and Razor Pages are added.
+
+-------------------
+
+<a name="hsts"></a>
+## HTTP Strict Transport Security Protocol (HSTS)
+
+
+Per [OWASP](https://www.owasp.org/index.php/About_The_Open_Web_Application_Security_Project), [HTTP Strict Transport Security (HSTS)](https://www.owasp.org/index.php/HTTP_Strict_Transport_Security_Cheat_Sheet) is an opt-in security enhancement that is specified by a web application through the use of a special response header. Once a supported browser receives this header that browser will prevent any communications from being sent over HTTP to the specified domain and will instead send all communications over HTTPS. It also prevents HTTPS click through prompts on browsers.
+
+ASP.NET Core 2.1 preview1 or later implements HSTS with the `UseHsts` extension method. The following code calls `UseHsts` when the app isn't in [development mode](xref:fundamentals/environments):
+
+[!code-csharp[sample](enforcing-ssl/sample/Startup.cs?name=snippet1&highlight=10)]
+
+`UseHsts` not recommend in development because calling `UseHsts` excludes the local loopback address.
```

Comment:
`UseHsts` not recommend in development because the HSTS header is highly cachable by browsers. By default, UseHsts excludes the local loopback address.